docs/userstories/v1/egress-control-in-anp/egress-control-outside-cluster.md

```diff
@@ -0,0 +1,56 @@
+
+### Summary
+
+1. **As a** cluster administrator **I want** to restrict traffic from
+specific cluster workloads to all or specific destinations outside the
+cluster **so that** I can enforce security for south-north traffic.
```

Comment:
```suggestion
cluster **so that** I can enforce security for northbound traffic.
```

The internet is at the top of the (notional) drawing, so egress is "north" and ingress (which we aren't handling here) is "south".

Reply:
yeah thanks Dan! `northbound` is definitely better; I confused it with south->(to)north haha, will change this.